zookeeper/src/test/java/com/linecorp/armeria/client/endpoint/zookeeper/TestBase.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.client.endpoint.zookeeper;
+
+import static org.junit.Assert.fail;
+
+import java.io.File;
+import java.time.Duration;
+import java.util.Set;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.collect.ImmutableSet;
+
+import com.linecorp.armeria.client.Endpoint;
+import com.linecorp.armeria.client.endpoint.zookeeper.common.Codec;
+import com.linecorp.armeria.client.endpoint.zookeeper.common.DefaultCodec;
+
+import zookeeperjunit.ZKFactory;
+import zookeeperjunit.ZKInstance;
+
+public class TestBase {
+    protected static final Logger logger = LoggerFactory.getLogger(EndpointGroupTest.class);
+    protected static final File ROOT_DIR = new File("build" + File.separator + "zookeeper");
```

Comment:
You can create a temporary directory and use it as the root directory.

```java
    @ClassRule
    public static final TemporaryFolder rootDir = new TemporaryFolder();
```

See [this article](https://garygregory.wordpress.com/2011/09/21/junit-4-9-to-the-rescue-with-classrule/) for an example: